test/unittest/dds/xtypes/type_representation/TypeObjectUtilsTests.cpp

```diff
@@ -3451,5 +3451,7 @@ int main(
         char** argv)
 {
     testing::InitGoogleTest(&argc, argv);
-    return RUN_ALL_TESTS();
+    int ret_value = RUN_ALL_TESTS();
+    eprosima::fastdds::dds::Log::KillThread();
```

Comment:
Should not this be done in a TearDown specialization? Only after one test is the issue appearing, not after running the complete suite.

Reply:
There is no TearDown without Fixtures. I will keep it in this way.